app-catalog/src/settings.tsx

```diff
@@ -0,0 +1,106 @@
+import { HoverInfoLabel, NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Box, Switch, SwitchProps, Typography, useTheme } from '@mui/material';
+import { useState } from 'react';
+import { store } from './components/charts/List';
+
+export function AppCatalogSettings() {
```

Comment:
AppCatalogSettings.tsx and AppCatalogSettings.stories.tsx in a separate commit.

Reply:
done!